02-cpu-configure/Makefile

```diff
@@ -0,0 +1,16 @@
+CFLAGS = -Wall -Werror -O2
+PREFIX ?= /usr/local
+
+all: pcie-remove pcie-rescan
+
+pcie-remove: pcie-remove.c
+	cc $(CFLAGS) $< -o $@
+
+pcie-rescan: pcie-rescan.c
+	cc $(CFLAGS) $< -o $@
+
+install: pcie-remove pcie-rescan
+	cp $? $(PREFIX)/bin
```

Comment:
Usual practice is to use `install` rather than `cp`, for no other reason than `cp` likes to reuse files, and might get you a "busy file" error or change an executable's `text` while it's running.

Reply:
done.